components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -82,11 +90,26 @@ export class ProjectListComponent implements OnInit, OnDestroy {
       }))
       .subscribe(() => {
         this.creatingProject = false;
+
+        const createTeamsWasChecked = this.createProjectForm.controls['addTeams'].value;
+        const projectID = this.createProjectForm.controls['id'].value;
+        const projectName = this.createProjectForm.controls['name'].value;
+
         this.closeCreateModal();
 
         // This is issued periodically from projects-filter.effects.ts; we do it now
         // so the user doesn't have to wait.
         this.store.dispatch(new LoadOptions());
+
+        if (createTeamsWasChecked) {
+          for (const [teamID, teamName] of Object.entries(this.teamsToCreate)) {
+            this.store.dispatch(new CreateTeam({
```

Comment:
I think to keep this component dumber, we'd want to handle this in an ngrx effect

Reply:
Hmmm I could see going either way on this one. Wanting to create a few teams as a result of how the user used the form feels like it makes sense to trigger in here. It feels weird that creating teams would be something the projects store would do.

On the other hand, I know we wanna keep these components as simple as possible within reason.